### ISSUE TITLE
Display Issues are warnings is not working

### ISSUE THREAD

**Issue by oliveregger** [ahdis/matchbox#178]
```json
{
  "resourceType" : "Observation",
  "id" : "d29b286d-b99c-410f-af4e-9455ea49d0af",
  "meta" : {
    "profile" : ["http://hl7.eu/fhir/laboratory/StructureDefinition/Observation-resultslab-eu-lab"]
  },
  "text" : {
    "status" : "generated",
    "div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Generated Narrative: Observation</b><a name=\"d29b286d-b99c-410f-af4e-9455ea49d0af\"> </a></p><div style=\"display: inline-block; background-color: #d9e0e7; padding: 6px; margin: 4px; border: 1px solid #8da1b4; border-radius: 5px; line-height: 60%\"><p style=\"margin-bottom: 0px\">Resource Observation &quot;d29b286d-b99c-410f-af4e-9455ea49d0af&quot; </p><p style=\"margin-bottom: 0px\">Profile: <a href=\"StructureDefinition-Observation-resultslab-eu-lab.html\">Observation Results: laboratory</a></p></div><p><b>status</b>: final</p><p><b>category</b>: Laboratory <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"http://terminology.hl7.org/5.3.0/CodeSystem-observation-category.html\">Observation Category Codes</a>#laboratory)</span></p><p><b>code</b>: Albumin (24H U) [Mass/Time] <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"https://loinc.org/\">LOINC</a>#1755-8 &quot;Albumin [Mass/time] in 24 hour Urine&quot;)</span></p><p><b>subject</b>: <a href=\"Bundle-SimpleChemistryResultReport.html#urn-uuid-de17bfd2-8d73-45fa-b0bb-8eb0e463ddb8\">urn:uuid:de17bfd2-8d73-45fa-b0bb-8eb0e463ddb8</a></p><p><b>effective</b>: 2023-03-27 11:24:26+0100</p><p><b>performer</b>: <span>: Jan Laborant</span></p><p><b>value</b>: 15 mg<span style=\"background: LightGoldenRodYellow\"> (Details: UCUM code mg = 'mg')</span>/24 h<span style=\"background: LightGoldenRodYellow\"> (Details: UCUM code h = 'h')</span></p></div>"
  },
  "status" : "final",
  "category" : [{
    "coding" : [{
      "system" : "http://terminology.hl7.org/CodeSystem/observation-category",
      "code" : "laboratory"
    }]
  }],
  "code" : {
    "coding" : [{
      "system" : "http://loinc.org",
      "code" : "1755-8",
      "display" : "Albumin [Mass/time] in 24 Stunden Urine"
    }],
    "text" : "Albumin (24H U) [Mass/Time]"
  },
  "subject" : {
    "reference" : "urn:uuid:de17bfd2-8d73-45fa-b0bb-8eb0e463ddb8"
  },
  "effectiveDateTime" : "2023-03-27T11:24:26+01:00",
  "performer" : [{
    "display" : "Jan Laborant"
  }],
  "valueRatio" : {
    "numerator" : {
      "value" : 15,
      "unit" : "mg",
      "system" : "http://unitsofmeasure.org",
      "code" : "mg"
    },
    "denominator" : {
      "value" : 24,
      "unit" : "h",
      "system" : "http://unitsofmeasure.org",
      "code" : "h"
    }
  }
}
```

<img width="805" alt="image" src="https://github.com/ahdis/matchbox/assets/2414516/5a69982b-3b38-4822-90b5-94152007a954">

1. There is no effect if you change it in the GUI in matchbox
2. You can also directly invoke it showing the behaviour:

```json
@host = https://test.ahdis.ch/matchboxv3/fhir
### @host = http://localhost:8080/matchboxv3/fhir
### @host = http://hapi.fhir.org/baseR4

## returns  "severity": "error", displayIssuesAreWarnings=true
POST {{host}}/$validate?profile=http://hl7.org/fhir/StructureDefinition/Basic HTTP/1.1
Accept: application/fhir+json
Content-Type: application/fhir+xml

<Basic xmlns="http://hl7.org/fhir">
    <code>
        <coding>
            <system value="http://loinc.org"/>
            <code value="1755-8"/>
            <display value="Albumin [Mass/time] in 24 Stunden Urine"/>
        </coding>
        <text value="Albumin (24H U) [Mass/Time]"/>
    </code>
</Basic>

### returns  "severity": "error", displayIssuesAreWarnings=true, Wrong Display Name 'Albumin [Mass/time] in 24 Stunden Urine' 
POST {{host}}/$validate?profile=http://hl7.eu/fhir/laboratory/StructureDefinition/Observation-resultslab-eu-lab HTTP/1.1
Accept: application/fhir+json
Content-Type: application/fhir+json

{
  "resourceType" : "Observation",
  "id" : "d29b286d-b99c-410f-af4e-9455ea49d0af",
  "meta" : {
    "profile" : ["http://hl7.eu/fhir/laboratory/StructureDefinition/Observation-resultslab-eu-lab"]
  },
  "text" : {
    "status" : "generated",
    "div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Generated Narrative: Observation</b><a name=\"d29b286d-b99c-410f-af4e-9455ea49d0af\"> </a></p><div style=\"display: inline-block; background-color: #d9e0e7; padding: 6px; margin: 4px; border: 1px solid #8da1b4; border-radius: 5px; line-height: 60%\"><p style=\"margin-bottom: 0px\">Resource Observation &quot;d29b286d-b99c-410f-af4e-9455ea49d0af&quot; </p><p style=\"margin-bottom: 0px\">Profile: <a href=\"StructureDefinition-Observation-resultslab-eu-lab.html\">Observation Results: laboratory</a></p></div><p><b>status</b>: final</p><p><b>category</b>: Laboratory <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"http://terminology.hl7.org/5.3.0/CodeSystem-observation-category.html\">Observation Category Codes</a>#laboratory)</span></p><p><b>code</b>: Albumin (24H U) [Mass/Time] <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"https://loinc.org/\">LOINC</a>#1755-8 &quot;Albumin [Mass/time] in 24 hour Urine&quot;)</span></p><p><b>subject</b>: <a href=\"Bundle-SimpleChemistryResultReport.html#urn-uuid-de17bfd2-8d73-45fa-b0bb-8eb0e463ddb8\">urn:uuid:de17bfd2-8d73-45fa-b0bb-8eb0e463ddb8</a></p><p><b>effective</b>: 2023-03-27 11:24:26+0100</p><p><b>performer</b>: <span>: Jan Laborant</span></p><p><b>value</b>: 15 mg<span style=\"background: LightGoldenRodYellow\"> (Details: UCUM code mg = 'mg')</span>/24 h<span style=\"background: LightGoldenRodYellow\"> (Details: UCUM code h = 'h')</span></p></div>"
  },
  "status" : "final",
  "category" : [{
    "coding" : [{
      "system" : "http://terminology.hl7.org/CodeSystem/observation-category",
      "code" : "laboratory"
    }]
  }],
  "code" : {
    "coding" : [{
      "system" : "http://loinc.org",
      "code" : "1755-8",
      "display" : "Albumin [Mass/time] in 24 Stunden Urine"
    }],
    "text" : "Albumin (24H U) [Mass/Time]"
  },
  "subject" : {
    "reference" : "urn:uuid:de17bfd2-8d73-45fa-b0bb-8eb0e463ddb8"
  },
  "effectiveDateTime" : "2023-03-27T11:24:26+01:00",
  "performer" : [{
    "display" : "Jan Laborant"
  }],
  "valueRatio" : {
    "numerator" : {
      "value" : 15,
      "unit" : "mg",
      "system" : "http://unitsofmeasure.org",
      "code" : "mg"
    },
    "denominator" : {
      "value" : 24,
      "unit" : "h",
      "system" : "http://unitsofmeasure.org",
      "code" : "h"
    }
  }
}
```

**Comment by qligier** [comment]
Could it be caused by the new TX server release?
In our integration tests, we also get errors while validation a code that is not within an extensible binding.

**Comment by oliveregger** [comment]
fixed with 3.8.4 (or before)
